WordPress/Classes/ViewRelated/NUX/LoginProloguePageViewController.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+class LoginProloguePageViewController: UIPageViewController {
+    var pages: [UIViewController] = []
+    fileprivate var pageControl: UIPageControl?
+    fileprivate var bgAnimation: UIViewPropertyAnimator?
+    fileprivate struct Constants {
+        static let topPadding: CGFloat = 20.0
+        static let pageControlHeight: CGFloat = 40.0
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        dataSource = self
+        delegate = self
```

Comment:
I wasn't able to set these in the storyboard—I don't think `self` is a valid target there ;)